### DISCUSSION TITLE
improve aider polyglot benchmark

### DISCUSSION THREAD

**Discussion by gengjiawen** [QwenLM/Qwen3#1139] [General]
https://aider.chat/docs/leaderboards/

currently Qwen2.5-Coder-32B-Instruct is only 8% completed correctly

Model | Percent completed correctly | Percent using correct edit format | Command | Edit format
-- | -- | -- | -- | --
o1-2024-12-17 | 61.7% | 91.5% | aider --model openrouter/openai/o1 | diff
claude-3-5-sonnet-20241022 | 45.3% | 100.0% | aider --model claude-3-5-sonnet-20241022 | diff
gemini-exp-1206 | 38.2% | 98.2% | aider --model gemini/gemini-exp-1206 | whole
o1-mini-2024-09-12 | 32.9% | 96.9% | aider --model o1-mini | whole
claude-3-5-haiku-20241022 | 28.0% | 91.1% | aider --model claude-3-5-haiku-20241022 | diff
gemini-2.0-flash-exp | 22.2% | 100.0% | aider --model gemini/gemini-2.0-flash-exp | whole
deepseek-chat | 17.8% | 92.9% | aider --model deepseek/deepseek-chat | diff
gpt-4o-2024-11-20 | 15.1% | 96.0% | aider --model gpt-4o-2024-11-20 | diff
Qwen2.5-Coder-32B-Instruct | 8.0% | 71.6% | aider --model openai/Qwen/Qwen2.5-Coder-32B-Instruct # via hyperbolic | diff
gpt-4o-mini-2024-07-18 | 3.6% | 100.0% | aider --model gpt-4o-mini-2024-07-18 | whole

**Comment by jklj077** [comment]
@huybery for visibility; please also consider posting at https://github.com/QwenLM/Qwen2.5-Coder/issues

**Reply by gengjiawen** [reply]
> @huybery for visibility; please also consider posting at [QwenLM/Qwen2.5-Coder/issues](https://github.com/QwenLM/Qwen2.5-Coder/issues?q=sort%3Aupdated-desc+is%3Aissue+is%3Aopen)

Actually I want Qwen2.5-72B-Instruct to be good on this. This is my goto qwen model
